modules/internal/Atomics.chpl

```diff
@@ -467,15 +467,15 @@ module Atomics {
       return ret;
     }
     inline proc add(value:uint(8), order:memory_order = memory_order_seq_cst):void {
-      on this do atomic_fetch_add_explicit_uint_least8_t(_v, value, order);
+      on this do local atomic_fetch_add_explicit_uint_least8_t(_v, value, order);
```

Comment:
I'm not sure what this is meant to be doing. This is adding a local block, not a local on.